connectors/src/connectors/google_drive/index.ts

```diff
@@ -528,6 +527,7 @@ export class GoogleDriveConnectorManager extends BaseConnectorManager<null> {
           },
         });
       } else if (permission === "read") {
+        addedFolderIds.push(id);
```

Comment:
If the permission was already read for that folder ID, this means we'll sync the folder again no ?

Reply:
no because we only get the delta from the call to setPermissions (i checked and tested)

Reply:
Nice !